test/powershell/engine/Basic/RegisterAssemblyResolverEarly.ps1

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "Assembly resolvers should be registered early at startup" -Tags "CI" {
+
+    ## The PKI module requires loading an assembly from GAC, and thus depends on the PowerShell assembly resolver to work.
+    It "Can load the PKI module with 'pwsh -ExecutionPolicy bypass -NoProfile -c `"Import-Module PKI`"'" -Skip:(!$IsWindows) {
+        $pwsh = Join-Path $PSHOME "pwsh.exe"
```

Comment:
This shouldn't be necessary since I believe we had a change so that $PSHOME is prepended to the path so that `pwsh` always starts an instance of the current pwsh

Reply:
Resolved. Now use `pwsh` directly.